### PR TITLE
ZENKO-3042 Fix list versions toggle

### PR DESCRIPTION
**Pull request by nicolas2bert**
- "List Versions" toggle should be disabled if bucket versioning is “Disabled”. 
- "List Versions" toggle should be enabled if bucket versioning is “Suspended“ or “Enabled“.

### CODE REVIEW

**Review by naren-rajendran — APPROVED** [review]

File: src/react/databrowser/objects/ObjectList.jsx [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 import * as L from '../../ui-elements/ListLayout2';
 import * as T from '../../ui-elements/Table';
+import type { BucketInfo, ListObjectsType, Object } from '../../../types/s3';
```
Comment (anurag4DSB) [inline src/react/databrowser/objects/ObjectList.jsx]: I checked `types/s3` to see what `Object` is.

Is it recommended/intentional to use the reserved `Object` ?

Comment (nicolas2bert) [reply src/react/databrowser/objects/ObjectList.jsx]: It was not intentional. ZenkoUI has a compiler to strip away Flow types.

Comment (nicolas2bert) [reply src/react/databrowser/objects/ObjectList.jsx]: I will change it to ObjectEntity for readability.

**Review by anurag4DSB — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e approve

**Pull request merged**